src/functions.php

```diff
@@ -133,13 +138,12 @@ function uploadImages(array $vcards, $config, $configPhonebook, callable $callba
     }
     ftp_close($ftp_conn);
 
-    error_log(PHP_EOL);
     if ($countAllImages > MAX_IMAGE_COUNT) {
-        error_log("WARNING: You have ".$countAllImages." contact images on FritzBox");
-        error_log("         FritzFon may handle only up to ".MAX_IMAGE_COUNT." images.");
-        error_log("         (see: https://github.com/andig/carddav2fb/issues/92)");
-        error_log("         Some images may not display properly.");
-        error_log("");
+        error_log(PHP_EOL."WARNING: You have ".$countAllImages." contact images on FritzBox");
```

Comment:
do we need the EOL inside error_log? If due to progress, only the very first one should be needed?

Reply:
I couldn't find the function definition of "error_log" (I admit, I didn't search very long) but you might be right, the first EOL should most likely be sufficient to "break" the progress bar. I'll remove the rest again.